code/modules/mob/living/simple_animal/hostile/glockroach.dm

```diff
@@ -67,3 +67,42 @@
 /mob/living/simple_animal/hostile/glockroach/ex_act() //Explosions are a terrible way to handle a cockroach.
 	return
 
+#define AS_MANY_BULLETS_A_MAG_GOT 15
+
+#define REAL_NIGGA_HOURS 24 HOURS
```

Comment:
brug

Reply:
bruh

Reply:
i can undefine it but it'll be like the zimmerman case part 2

Reply:
change it before github police finds it